wherehows-web/app/components/dataset-compliance.ts

```diff
@@ -565,20 +574,6 @@ export default class DatasetCompliance extends Component {
     get(this, 'foldChangeSetTask').perform();
   }
 
-  /**
-   * Parent task to determine if a compliance policy can be created or updated for the dataset
-   * @type {Task<TaskInstance<Promise<Array<IDataPlatform>>>, () => TaskInstance<TaskInstance<Promise<Array<IDataPlatform>>>>>}
-   * @memberof DatasetCompliance
-   */
-  complianceAvailabilityTask = task(function*(
```

Comment:
ps remove reference to this task from https://github.com/linkedin/WhereHows/blob/f3245032d0075f24ba1322ac9b804e27df317e77/wherehows-web/app/templates/components/dataset-compliance.hbs#L1